src/ray/protobuf/core_worker.proto

```diff
@@ -218,6 +218,12 @@ message WaitForRefRemovedReply {
   repeated ObjectReferenceCount borrowed_refs = 1;
 }
 
+message LocalGCRequest {
```

Comment:
```suggestion
message TriggerLocalGCRequest {
```

Reply:
I probably won't change this, it doesn't seem that much more clear.

Reply:
That's fine, just bothers me because the other RPCs start with a verb